reference/6/Microsoft.PowerShell.Core/Enter-PSSession.md

```diff
@@ -304,6 +305,9 @@ Specifies the session configuration that is used for the interactive session.
 Enter a configuration name or the fully qualified resource URI for a session configuration.
 If you specify only the configuration name, the following schema URI is prepended: http://schemas.microsoft.com/powershell.
 
+When used with SSH, this specifies the subsystem to use on the target as defined in sshd_config.
```

Comment:
I feel that ConfigurationName parameter should not be used to specify an SSH session subsystem.  We should reserve this for when we add session configuration support to SSH.